core/state_transition.go

```diff
@@ -155,6 +164,15 @@ func IntrinsicGas(data []byte, accessList types.AccessList, isContractCreation b
 	return gas, nil
 }
 
+// toWordSize returns the ceiled word size required for init code payment calculation.
+func toWordSize(size uint64) uint64 {
```

Comment:
Isn't there a helper for this already?

Reply:
There is, but private in evm package.

Reply:
@holiman should this help in the evm package made public?

Reply:
IMO not necessarily, it's such a small thing we  might live with some copy-pasta